guidelines/templates/resources/custom-resource.md

```diff
@@ -0,0 +1,40 @@
+---
+title: {CRD kind}
+type: Custom Resource
+---
+
+> This document is a ready-to-use template for a custom resource (CR) document that provides a sample custom resource and description of its elements. Additionally, the document points to the Custom Resource Definition (CRD) used to create CRs of the given kind. For reference, see the existing documents for the [Installation](https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/040-installation-custom-resource.md) and the [Api](https://github.com/kyma-project/kyma/blob/master/docs/api-gateway/docs/011-api-custom-resource.md) CRs.
```

Comment:
custom resource -> capitalize 

:D

Reply:
It was my advice to not capitalize as it is not an invented name (however `nazwa wlasna` is called in english in this context). Is is just a resource, but custom one. Custom Resource Definition is an invented name
https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/